extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.settings.js

```diff
@@ -0,0 +1,202 @@
+define('wikia.articleVideo.featuredVideo.jwplayer.plugin.settings', ['wikia.articleVideo.featuredVideo.autoplay'], function (featuredVideoAutoplay) {
+
+	function WikiaJWPlayerSettings(player, config, div) {
+		this.player = player;
+		this.wikiaSettingsElement = div;
```

Comment:
please rename `div` to `wrapper` or `element`

Reply:
what's wrong with `div`? :)

Reply:
are we sure it'll always be `div`? 🙈 

you can leave it, no problem